composite-view/src/test/java/com/illuwatar/compositeview/JavaBeansTest.java

```diff
@@ -0,0 +1,59 @@
+package com.illuwatar.compositeview;
```

Comment:
The file path should be `composite-view/src/test/java/com/iluwatar/compositeview/JavaBeansTest.java`

Reply:
That is the file path I have right now.

Reply:
Tests are in the wrong package/directory name. One "el" not two.

> `composite-view/src/test/java/com/illuwatar/compositeview/JavaBeansTest.java`

should be

> `composite-view/src/test/java/com/iluwatar/compositeview/JavaBeansTest.java`

and package should be `com.iluwatar.compositevie` not `com.illuwatar.compositeview`

Reply:
ah I see, thanks